src/pages/networks/NetworkForwardCount.tsx

```diff
@@ -0,0 +1,21 @@
+import React, { FC } from "react";
+import { useQuery } from "@tanstack/react-query";
+import { queryKeys } from "util/queryKeys";
+import { LxdNetwork } from "types/network";
+import { fetchNetworkForwards } from "api/network-forwards";
+
+interface Props {
+  network: LxdNetwork;
+  project: string;
+}
+
+const NetworkForwardCount: FC<Props> = ({ network, project }) => {
+  const { data: forwards = [] } = useQuery({
```

Comment:
maybe worthwhile to put a loader in this component to improve visuals instead of having numbers go from 0 -> count after some time?

Reply:
I think a loader would be too distracting, but we can keep it empty until the loading is finished.

Reply:
sounds good :+1: